java/src/jmri/jmrix/can/cbus/CbusBundle.properties

```diff
@@ -993,5 +993,7 @@ ProgArMode = Programming track is Auto-reverse district
 HardwareModeLabel = Select the hardware operating mode.
 NoCbusProgrammer = No Cbus DCC Programmer available
 HideFurtherWarnings = Hide further warnings
+ProgWarning = Close any programmer when changing modes
+switchMode = PLease Close Programmer
```

Comment:
Typo: PLease -> Please.  Also maybe "Please close programmer" for consistent capitalization with other messages